draft-ietf-httpbis-priority.md

```diff
@@ -175,6 +174,10 @@ to more complicated processing. Problems found with this approach could not be
 resolved and definition of a HTTP/3 priority signalling feature was removed
 before publication.
 
+RFC 7540 priorities were specific to a HTTP/2 connection and could not be
+conveyed with prior versions of HTTP.  That, along with the complexity, meant
+intermediaries could not convey the priority information to backends.
```

Comment:
could not or could not easily?

That answer depends on the nature the assets being requested, the intermediary and the backend. For instance, where a backend is in the same adiministrative domain, pretty much anything is possible. For example, https://developers.cloudflare.com/workers/runtime-apis/request#properties describes the `request.cf.requestPriority` object that is available in on type of context.

So I think maybe what we mean is that there is/was no standard format for exchanging RFC 7540 priority information in prior versions of HTTP (or inded newer versions). Therefore I might turn this paragraph around like so

```suggestion
Intermediaries can convert HTTP/2 to other versions of HTTP. HTTP/2 priority signals are specific to a connection and there is no standard definition for converting them to a format suitable for other versions. Therefore, a motivating factor for replacing RFC 7540 stream priority is to choose a signalling mechanism that is more accomodating of intermediaries.
```